build.gradle

```diff
@@ -1,7 +1,7 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.gradle_version = '3.5.2'
+    ext.gradle_version = '3.6.0-alpha04'
```

Comment:
its better not to use alpha versions. they can be buggy and waste development time. the last non-alpha version is '3.6.0'